recipe/meta.yaml

```
@@ -15,9 +15,11 @@
 # tbb4py files should be installed into PREFIX.
 # See https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html.
 # On non-Windows CMAKE_ARGS already contains CMAKE_INSTALL_PREFIX.
-{% set cmake_args = "$CMAKE_ARGS" %}                                                         # [not win]
+{% set cmake_args = "$CMAKE_ARGS -DCMAKE_INSTALL_LIBDIR=lib" %}  # [not win] 
+{% set cmake_args = "$CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX" %}  # [not win]                                                       # [not win]
```

We need to be careful here with what variables we use and the order. In this case, the **CMAKE_INSTALL_LIBDIR** is not set properly because the next line is overwriting the `cmake_args` variable, not appending to it.

Without this set, the libs are likely installed to `lib64/`, instead of the expected `lib/`

```suggestion
{% set cmake_args = "$CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib" %}  # [not win]
```


Additionally, we might want to only set these for the platforms that need them. Generally, these consistent args should be set by the _updated_ compiler activation packages.